bin/generate.js

```diff
@@ -60,6 +60,9 @@ if (args.h || args.help || args._.length > 1) {
     // Override the baseURL in the webpack config
     webpackConfig.baseURL.replace = baseURL;
 
+    // Override the buildContext in the webpack config
+    webpackConfig.buildContext.dir = destinationPath;
```

Comment:
While I certainly like the effort you did you create a custom Webpack plugin, I wonder if it would not be sufficient to just configure a different `outputFile` when calling `compileSettings`? https://github.com/comunica/jQuery-Widget.js/blob/f72df7e8090bce15e2fae6c6a76f032ff55d2674/bin/generate.js#L52
Unless you see a good reason not to do this?

Reply:
I considered using a different `outputFile` when calling `compileSettings`, as you suggested. However, I opted for the custom Webpack plugin to ensure that we do not depend on pre-existing files in the chosen build directory. As Webpack should ideally clean this chosen build directory first before creating the new build to avoid conflicts with older build files. 

The custom plugin approach would allow this cleaning to happen within Webpack, so all logic related to creating and populating the build folder remains consistent.

Reply:
Ok, that's a very good point indeed!